doc/release-notes-1.0.0.rst

```diff
@@ -0,0 +1,323 @@
+
+The following sections provide detailed lists of changes by component.
+
+nRF9160
+=======
+
+* Added the following samples:
+
+  * :ref:`aws_fota_sample` - shows how to perform over-the-air firmware updates of an nRF9160 via MQTT and HTTP.
+  * :ref:`secure_services` - demonstrates using the reboot and random number services.
+
+* Added the following libraries:
+
+  * :ref:`lib_fota_download` - handles Firmware Over The Air (FOTA) downloads.
+  * :ref:`lib_aws_fota` - combines the :ref:`lib_aws_jobs` and :ref:`lib_fota_download` libraries to create a user-friendly library that can perform firmware-over-the-air (FOTA) update using HTTP and MQTT TLS.
+  * :ref:`at_cmd_readme` - facilitates handling of AT Commands by multiple modules.
+  * :ref:`lib_aws_jobs` - facilitates communication with the AWS IoT Jobs service.
+
+* Asset Tracker sample:
+
+  * The orientation detector now supports interrupt handling.
+
+* nRF Connect SDK now uses upstream CoAP implementation. The :ref:`mqtt_simple_sample` sample was rewritten to use the upstream library, and the downstream CoAP was removed.
+* The :ref:`http_application_update_sample` sample has been updated to use the :ref:`lib_fota_download` library.
+
+BSD library
+-----------
+
+* Updated bsdlib to version 0.3.1.
+* Introduced a new header bsdlib.h to be used by the application to initialize and shutdown the library.
+* Library initialization during system initialization (``SYS_INIT``) is now optional, and controlled via ``Kconfig``. The default behavior is unchanged.
+
+Secure Partition Manager (SPM) library
+--------------------------------------
+
+* Added random number secure service, providing access to the RNG hardware from the non-secure firmware.
+* Non-Secure callable support:
+
+  * A secure_services module is now available over secure entry functions. This means:
+
+    * :file:`secure_services.c` resides in secure firmware (SPM).
+    * :file:`secure_services.h` declares functions that can be called from non-secure firmware.
+
+  * SPM now exposes secure entry functions by default.
+  * Added reboot as a secure service. The reboot secure service is called when the non-secure firmware calls sys_reboot().
+
+* PWM0-3 added as non-secure.
+
+
+Common libraries
+================
+
+* Added the following library:
+
+  * :ref:`ppi_trace` - enables tracing of hardware peripheral events on pins.
+
+Enhanced Shockburst
+-------------------
+
+* Added support for nRF52811.
+
+Download Client
+---------------
+
+* Added IPv6 support, with fallback to IPv4.
+* Added HTTPS support. The application must provision the TLS security credentials.
+* Several improvements to buffer handling and network code.
+* Library now runs in a separate thread.
+
+
+Crypto
+======
+
+* Added :ref:`nrfxlib:nrf_cc310_mbedcrypto_readme` library v0.8.1 (experimental release) to perform hardware-accelerated cryptography using Arm CryptoCell CC310 in selected architectures.
+
+nRF BLE Controller
+==================
+
```

Comment:
Should there be a reference to the changelog of the ble controller?

Reply:
sounds good, done